GeoLib/IO/XmlIO/Boost/BoostXmlGmlInterface.cpp

```diff
@@ -42,7 +42,6 @@ bool BoostXmlGmlInterface::readFile(const std::string &fname)
 
     // ignore attributes related to XML schema
     doc->ignoreConfigAttribute("xmlns:xsi");
-    doc->ignoreConfigAttribute("xsi:noNamespaceSchemaLocation");
```

Comment:
Removing that line will probably lead to errors with existing files that contain that attribute. Maybe better leave in order to not frustrate users?

Reply:
Good point, thanks!